src/main/java/edu/ucla/library/prl/harvester/services/HarvestScheduleStoreServiceImpl.java

```diff
@@ -82,7 +143,39 @@ public Future<Void> removeJob(final int aJobId) {
 
     @Override
     public Future<Void> close() {
-        // TODO implement method
-        return Future.succeededFuture(null);
+        return myDbConnectionPool.close();
+    }
+
+    /**
+     * Gets the database's connection options.
+     *
+     * @param aConfig A configuration
+     * @return The database's connection options
+     */
+    private PgConnectOptions getConnectionOpts(final JsonObject aConfig) {
```

Comment:
If we go with the one user approach, do we need all these? Couldn't we still just use the ENVs to automatically configure the DB?

Reply:
Yes, we need this... tried to do just `PgPool.pool` and got the following error:
```
[ERROR] io.vertx.core.impl.ContextBase | Unhandled exception
java.lang.IllegalStateException: Running in a Vertx context => use Pool#pool(Vertx, SqlConnectOptions, PoolOptions) instead
```

Reply:
Not sure I understand. What's different between here and https://github.com/UCLALibrary/prl-harvester/blob/14d85d1f45ce7d41ba6c6c123f2ae3bfb178ec0c/src/test/java/edu/ucla/library/prl/harvester/DatabaseIT.java#L76 which works?

Reply:
Okay, this shouldn't hold us up. I'll investigate at a later date.